cluster-agent/controllers/argoproj.io/application_controller_test.go

```diff
@@ -411,6 +411,77 @@ var _ = Describe("Application Controller", func() {
 			Expect(result).ToNot(BeNil())
 
 		})
+
+		It("Test to check whether reconciled state is stored in database", func() {
```

Comment:
FYI the unit tests appear to be failing in the CI job, in these functions.

Reply:
I will check this @jgwest I did some rebase and forgot top test it properly again